packages/fetchai/connections/p2p_libp2p_client/connection.py

```diff
@@ -58,6 +58,131 @@
 ACN_CURRENT_VERSION = "0.1.0"
 
 
+class NodeClient:
+    """Client to communicate with node using ipc channel(pipe)."""
+
+    def __init__(self, pipe: IPCChannelClient, logger: logging.Logger) -> None:
+        """Set node client with pipe."""
+        self.pipe = pipe
+        self._buffer = b""
+        self.logger = logger
```

Comment:
WE don't have  buffer and logger in the p2p_libp2p, probably make it the same there?

Reply:
buffer is an old part of code i removed.
logger can be removed as well and own logger used